app/src/main/java/io/apicurio/registry/rest/beans/Rule.java

```diff
@@ -3,7 +3,9 @@
 
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyDescription;
 import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import io.apicurio.registry.types.RuleType;
```

Comment:
Hmmm, wrt my previous comment, does this mean we have duplicate RuleType class?

Reply:
Looking again at the diff I see there is only one `RuleType` enum generated (which is good) but somehow one of the imports got messed up.  I'll check it out.